.github/actions/kfp-cluster/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
   k8s_version:
     description: "The Kubernetes version to use for the Kind cluster"
     required: true
+  publish_logs:
```

Comment:
@hbelmiro tysm for providing a reference implementation. I sincerely hope this doesn't clobber your PR with merge conflicts.